OJS.Workers.ExecutionStrategies/RunSpaAndExecuteMochaTestsExecutionStrategy.cs

```diff
@@ -205,13 +263,14 @@ class DockerExecutor:
                 return result;
             }
 
-            this.SaveTestsToFiles(executionContext.Input.Tests);
+            // this.SaveTestsToFiles(executionContext.Input.Tests);
             this.SaveNginxFile();
 
-            var codeSavePath = this.SavePythonCodeTemplateToTempFile();
+            this.PreExecuteCodeSavePath = this.SavePythonCodeTemplateToTempFile(this.PythonPreExecuteCodeTemplate);
+            // var codeSavePath = this.SavePythonCodeTemplateToTempFile(this.PythonCodeTemplate);
             var executor = this.CreateExecutor();
             var checker = executionContext.Input.GetChecker();
-            return this.RunTests(codeSavePath, executor, checker, executionContext, result);
+            return this.RunTests(string.Empty, executor, checker, executionContext, result);
```

Comment:
Do not pass string.Empty
Change method structure instead

Reply:
Changed it to pass the pre execute code save path

Reply:
After latest changes, I need to leave it as empty again, since the actual execution code depends on test information and will be modified for executing on each test. The RunTests method is inherited, so I can't change it, is it better to create a new method and not use the inherited one?